packages/components/src/components/textarea/textarea.examples.md

```diff
@@ -0,0 +1,183 @@
+<ComponentHeading name="Textarea"></ComponentHeading>
+
+The `p-textarea` component is a multi-line text input control. Don't use a Textarea component if you want to allow users
+to enter shorter responses that are no longer than a single line, such as a phone number or name. In this case, you
+should use the Text Field component.
+
+A `label` is a caption which informs the user what information a particular form field is asking for. The `p-textarea`
+component can be used with or without a label, but it's recommended to keep the label visible for better accessibility
+whenever possible. When used without a label, it's best practice to provide a descriptive label text for screen readers.
```

Comment:
"~~it's best practice~~" should be "it's mandatory"

Reply:
updated